### PR TITLE
Add ChangeLog_Pending

### PR DESCRIPTION
**Pull request by marbar3778**
- Increase amount of contributions to this repo brings the need of a changelog_pending file.

Signed-off-by: Marko Baricevic <marbar3778@yahoo.com>

### CODE REVIEW

File: CHANGELOG_PENDING.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+\*\*
+
+Special thanks to external contributors on this release:
+
+### Changes
+
+- Implement LazyLoadVersion(@alexanderbez)
+- Move to go.mod(@Liamsi)
+- Iaviewer command to visualize iavl database from leveldb(@ethanfrey)
```
Comment (fedekunze) [inline CHANGELOG_PENDING.md]:
```suggestion
- `Iaviewer` command to visualize iavl database from leveldb(@ethanfrey)
```

File: CHANGELOG_PENDING.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+\*\*
+
+Special thanks to external contributors on this release:
+
+### IMPROVEMENTS
+
+- Implement LazyLoadVersion (@alexanderbez)
+  - LazyLoadVersion attempts to lazy load only the specified target version
+    without loading previous roots/versions. - see [goDoc](https://godoc.org/github.com/tendermint/iavl#MutableTree.LazyLoadVersion)
+- Move to go.mod(@Liamsi)
```
Comment (melekes) [inline CHANGELOG_PENDING.md]:
```suggestion
- Move to go.mod (@Liamsi)
```

File: CHANGELOG_PENDING.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+- `Iaviewer` command to visualize iavl database from leveldb(@ethanfrey)
```
Comment (melekes) [inline CHANGELOG_PENDING.md]:
```suggestion
- `iaviewer` command to visualize IAVL database from leveldb (@ethanfrey)
```

File: CHANGELOG_PENDING.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+## 0.12.3
+
+\*\*
+
+Special thanks to external contributors on this release:
+
+### IMPROVEMENTS
+
+- Implement LazyLoadVersion (@alexanderbez)
+  - LazyLoadVersion attempts to lazy load only the specified target version
```
Comment (melekes) [inline CHANGELOG_PENDING.md]: remove the `-`

**Pull request merged**